Makefile

```diff
@@ -1,6 +1,6 @@
 CC := $(CC)
 CXX := $(CXX)
-CXXFLAGS := $(CXXFLAGS) -Iinclude -Imason_packages/.link/include -std=c++11
+CXXFLAGS := $(CXXFLAGS) -Iinclude -Imason_packages/.link/include -I/usr/local/include -std=c++11
```

Comment:
Aha, you likely hit a problem with missing boost because the makefile was too simplistic and would not install boost if geometry was already there. Fixed in 62d825550a07f7795690f495a79124beb489a0e0 and therefore I removed the `/usr/local/include` in 8a698be577aa713c06135f680c73d799cc50a428